docs/can-guides/topics/data/managing-sessions/managing-sessions.md

```diff
@@ -156,8 +156,13 @@ After the logout completes `Session.current` will be set to undefined and `Sessi
 The following is the full example of using cookie-based sessions:
 
 @sourceref ./cookie-session-vm.html
+@highlight 16-31, 44, 52, 56, 91, 98-101, 103-105, 124-131, only  
 @codepen 
 
+<p style="text-align: center">
+<img src="../../docs/can-guides/images/managing-sessions/example.gif" style="width:100%; max-width:745px;" />
+</p>
```

Comment:
You can use the `bit-docs-screenshot ` class on the image. Here’s more info: https://github.com/canjs/canjs/pull/5183